index.md

```diff
@@ -7,60 +7,46 @@ permalink:
 
 # Hello Decentralized World
 {: .fs-8 }
-<img style="float: right; padding:10px;" src="assets/images/helloworldsquare.png">
+<img style="float:right; padding:6px; width:529px;" src="assets/images/wasm_sample_code.png">
```

Comment:
I think ideally this would not be an image, but rather a div with a <code> tag in it.

Reply:
Agreed.  It would be better as code and then even "cut and pasteable".
I will look into that.